save-common/src/nativeMain/kotlin/org/cqfn/save/core/utils/ProcessBuilderInternal.kt

```diff
@@ -12,12 +12,12 @@ import platform.posix.system
 actual class ProcessBuilderInternal actual constructor(
     private val stdoutFile: Path,
     private val stderrFile: Path,
-    private val collectStdout: Boolean
+    private val useInternalRedirections: Boolean
 ) {
-    actual fun prepareCmd(command: String) = if (collectStdout) {
-        "$command >$stdoutFile 2>$stderrFile"
+    actual fun prepareCmd(command: String) = if (useInternalRedirections) {
+        "($command) >$stdoutFile 2>$stderrFile"
```

Comment:
We need brackets here, since if command is complex (contain `&&` or `;`) we need to redirect output from all command at the one time.

The minus of this approach - now we can't redirect output, if command already have redirections.
`(cmd > /dev/null)` > `>$stdoutFile 2>$stderrFile` - redirection to `/dev/null` will applied at any time.
However, we don't allow user redirections, so I think it's okay. We use redirections in cmd only for tests